webapp/packages/core-extensions/src/ExtensionUtils.ts

```diff
@@ -13,6 +13,7 @@ interface ExtensionExecutor {
     predicate: (extension: IExtension<any>) => extension is T,
     action: (extension: T) => void
   ) => this;
+  has: (checker: (extension: IExtension<any>) => boolean) => boolean;
```

Comment:
wrong type definition for extension predicate function (see type for predicate in `on`)